src/freertos_drivers/spiffs/stm32f0_f3/Stm32SPIFFS.cxx

```diff
@@ -0,0 +1,143 @@
+/** @copyright
+ * Copyright (c) 2018, Stuart W Baker
```

Comment:
This and other new files should be attributed to Balazs as the author and copyright holder, also update the dates.

Reply:
done